extensions/amp-a4a/0.1/test/test-real-time-config-manager.js

```diff
@@ -104,7 +104,7 @@ describes.realWin('real-time-config-manager', {amp: true}, env => {
         expect(rtcResponseArray.length).to.equal(expectedRtcArray.length);
         expect(fetchJsonStub.callCount).to.equal(calloutCount);
         (expectedCalloutUrls || []).forEach(url => {
-          expect(fetchJsonStub.calledWith(url));
+          expect(fetchJsonStub.calledWith(url)).to.be.true;
```

Comment:
This should be:
```
expect(fetchJsonStub).to.have.been.calledWith(url);
```

At that point, you might have to disable the test `'should favor publisher URLs over vendor URLs'` which appeared to be silently failing in the past.